### PR TITLE
Integrate Google Analytics 4

### PR DESCRIPTION
**Pull request by LuSilvaDeveloper**
|                                    Web Dev Path                                    |
| :--------------------------------------------------------------------------------: |
| [**177**](https://github.com/Web-Dev-Path/web-dev-path/issues/177) |

#### Have you updated the CHANGELOG.md file? If not, please do it.

Yes.

#### What is this change?

Integrated Google Analytics 4 (GA4) using the library next-ga.

#### Were there any complications while making this change?

No.

#### How did you verify this change?

I tested it locally using many browsers, such as Chrome, Firefox, Edge, and Safari (MAC).

#### When should this be merged?

After review

### CODE REVIEW

**Comment by mariana-caldas** [comment]
Hey @LuSilvaDeveloper !

Great job on submitting your first PR 🎉 

Can you please do the following steps?
1.  delete the `.yarn` folder with your local yarn configurations, 
2. add the following to our `.gitignore` since we don't want to version control your local cache.

```
.yarn/*
!.yarn/releases
!.yarn/plugins
!.yarn/sdks
```
3. commit the change

https://stackoverflow.com/questions/69885538/should-i-push-the-yarn-cache-folder-to-github-yarn2

Please let me know if you have any questions!

**Comment by LuSilvaDeveloper** [comment]
Hey @mariana-caldas .
Thank you!
I've already fixed it.

File: package.json [hunk 1]
```diff
@@ -20,8 +20,10 @@
   },
   "dependencies": {
     "@sendgrid/mail": "^7.7.0",
+    "dotenv": "^16.3.1",
```
Comment (cherylli) [inline package.json:23]: I believe dotenv is not required for next.js

File: pages/_app.js [hunk 2]
```diff
@@ -1,5 +1,10 @@
 import Layout from '@/components/layout/Layout';
 import '@/styles/globals.scss';
+import withGA from 'next-ga';
+import Router from 'next/router';
+
+require('dotenv').config();
```
Comment (cherylli) [inline pages/_app.js:6]: as above, dotenv is not requried for next.js

**Review by cherylli — COMMENTED** [review]
Hi, thanks for the PR, just left a few comments, I'm still waiting for the key to test out the app locally

**Review by mariana-caldas — CHANGES_REQUESTED** [review]
Other than the comments Cheryl mentions about the "dotenv" unnecessary library, I think we're good to!

File: pages/_app.js (hunk 2 above)
Comment (LuSilvaDeveloper) [reply pages/_app.js]: alright. Thank you! I've already uninstalled the dotenv library.

**Review by sergiobtos — APPROVED** [review]
Looks good. All the possible changes were already made by the team.

**Review by cherylli — APPROVED** [review]
Looks good!

**Review by mariana-caldas — APPROVED** [review]
Great job, @LuSilvaDeveloper !

**Pull request merged**